upgrade/breaking-changes/4-0-0 .md

```diff
@@ -0,0 +1,308 @@
+            <td style="vertical-align:top">
+<pre><code>
+&lt;SchedulerPopupEditFormSettings ButtonsLayout="FormButtonsLayout.Stretch" /&gt;
+</code></pre>
+            </td>
+        </tr>
+        <tr>
+            <td style="vertical-align:top">
+<pre><code>
+&lt;TreeListPopupEditFormSettings ButtonsLayout="FormButtonsLayout.Stretched" /&gt;
+</code></pre>
+            </td>
+            <td style="vertical-align:top">
+<pre><code>
+&lt;TreeListPopupEditFormSettings ButtonsLayout="FormButtonsLayout.Stretch" /&gt;
+</code></pre>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### Async Method Suffix
+
+Added the `Async` suffix to the signature of asynchronous methods
+
+>caption Method signatures in UI for Blazor up to version 3.8.0 and after version 4.0.0
+
+<table>
+    <thead><tr>
+        <th>UI for Blazor 3.8.0</th>
+        <th>UI for Blazor 4.0.0</th>
+    </tr></thead>
+    <tbody>
+        <tr>
+            <td style="vertical-align:top">
+<pre><code>
+await TelerikGridReference.SetState(desiredState);
+</code></pre>
+            </td>
+            <td style="vertical-align:top">
+<pre><code>
+await TelerikGridReference.SetStateAsync(desiredState);
+</code></pre>
+            </td>
+        </tr>
+        <tr>
+            <td style="vertical-align:top">
+<pre><code>
+await TelerikTreeListReference.SetState(desiredState);
+</code></pre>
+            </td>
+            <td style="vertical-align:top">
+<pre><code>
+await TelerikTreeListReference.SetStateAsync(desiredState);
+</code></pre>
+            </td>
+        </tr> 
+    </tbody>
+</table>
+
+### Obsolete Parameters
+
+* Removed the `DefaultItem` parameter from the Telerik DropDownList.
+
+## Component Changes
+
+### Button
+
+- Removed the `Shape` parameter.
+- Changed the signature of the `Hidden` parameter to `Visible`.
+
+>caption Visible parameter in UI for Blazor up to version 3.8.0 and after version 4.0.0
+
+<table>
+    <thead><tr>
+        <th>UI for Blazor 3.8.0</th>
+        <th>UI for Blazor 4.0.0</th>
+    </tr></thead>
+    <tbody>
+        <tr>
+            <td style="vertical-align:top">
+<pre><code>
+&lt;TelerikButton Hidden="false"&gt; Visible button &lt;TelerikButton /&gt;
+</code></pre>
+            </td>
+            <td style="vertical-align:top">
+<pre><code>
+&lt;TelerikButton Visible="true"&gt; Visible button &lt;TelerikButton /&gt;
+</code></pre>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### ButtonGroup
+
+- Removed the `Shape` parameter.
+
+### Drawer
+
+- Changed the values in the `DrawerPosition` enum from `Left` and `Right` to `Start` and `End`.
+
+>caption DrawerPosition enum values in UI for Blazor up to version 3.8.0 and after version 4.0.0
+
+<table>
+    <thead><tr>
+        <th>UI for Blazor 3.8.0</th>
+        <th>UI for Blazor 4.0.0</th>
+    </tr></thead>
+    <tbody>
+        <tr>
+            <td style="vertical-align:top">
+<pre><code>
+&lt;TelerikDrawer Position="@DrawerPosition.Left/Right"&gt;
```

Comment:
Let's use valid code snippets and count that our customers will fill in the blanks.

```suggestion
&lt;TelerikDrawer Position="@DrawerPosition.Left"&gt;
```